src/components/notifications/notifications.jsx

```diff
@@ -0,0 +1,18 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import styles from './notifications.module.css';
+
+export const Notification = ({ className, Icon, label, color }) => {
+    return (
```

Comment:
```suggestion
    const Icon = icons[type];
    
    return (
```

ту миотримуємо потрібну іконку в залежності від `type` (success, warning або error)